### DISCUSSION TITLE
Feature request: Don't want to see activity from people I've collaborated with

### DISCUSSION THREAD

**Discussion by avegancafe** [community/community#38305] [Other Feature Feedback, Questions, & Ideas]
As the title says. My feed has quite a lot of noise that makes it uninteresting and ignorable for me when I use github. For example, I don't want to see random user activity from users I've happened to collaborate with in the past decade. This seems like a recipe for this being very noisy for anyone working in open source. Not only that, there's not even an option to not see that kind of activity if I don't want to. Is there any other way to completely disable this type of activity from showing up in my feed? 

<img width="1411" alt="image" src="https://user-images.githubusercontent.com/2652762/200051749-c7b0965f-c15b-4040-ab14-80ed30aff4f1.png">

**Comment by david-engelmann** [comment]
[I've seen similar request in the past](https://github.com/community/community/discussions/31863). As of now, I believe they can clear all your activity if you email them but you can't remove certain topics or people. Even earlier today, [someone mentioned](https://github.com/community/community/discussions/38294) the clutter from having contributed to a repo that has been forked many times.

**Comment by ossanna16** [comment]
Hi there @avegancafe and welcome to our community! Thank you for the feedback 🙂

**Comment by nunziotocci** [comment]
I'd like to at least be able to see *when* GH says I collaborated with someone. For example, I have no idea who these people are:
![image](https://user-images.githubusercontent.com/7560749/211911813-a5b595a8-7c6e-4232-8c32-252e70292b20.png)

Being able to hide "collaborated" entries outright would be great too. It just clutters up my feed.

**Reply by avegancafe** [reply]
Same lol. I've worked in so many random github organizations over the years, e.g. I get updates from basically everyone I went to college with because we were in a github org together. Very useless, unfortunately.
